src/controller/nwnx.ini

```diff
@@ -84,3 +84,9 @@ gracefulShutdownMessage = Server is shutting down NOW!
 # Value = seconds to wait before actual shutdown
 # default: 5 seconds
 gracefulShutdownMessageWait = 5
+
+#logServiceUrl=
```

Comment:
Some documentation for these values would be nice :)

Reply:
Will do.